docker/development/docker-compose.yml

```diff
@@ -134,7 +134,32 @@ services:
       - redis
       - mailcatcher
 
+  nginx:
+    image: openresty/openresty:alpine
+    # Until https://github.com/nginx-proxy/docker-gen/pull/311 is merged, use hardcoded name
+    container_name: mampf-docker-nginx
+    ports:
+      - "127.0.0.1:3000:80"
+    volumes:
+      - "nginx-conf-d:/etc/nginx/conf.d"
+      - "submissions:/private/submissions:ro,nocopy"
```

Comment:
What is the `submission` volume?

Reply:
Submissions are exercise sheets submitted by students. The need to be handled by nginx, because they are served there. Nginx still asks rails for permissions on those files and rails answers this without sending the actual file contents. This is called accelerated redirect and is done on the server anyway.